README.md

```diff
@@ -79,11 +79,22 @@ Add deep link support to your app in 5 minutes or less following these simple st
 <br />
 **4. Register a route handler**
 
+Objective-C
 ````objc
 self.router[@"/log/:message"] = ^(DPLDeepLink *link) {
   NSLog(@"%@", link.routeParameters[@"message"]);
 };
 ````
+
+Swift
+````swift
+self.router.registerRoute("/log/:message") { link in
```

Comment:
This would ideally be changed to come through as `register(route:)`

Reply:
It doesn't because there's already a method called `-registerBlock:forRoute:` which gets the shorthand `register:`. Changing the order in the header would do it but also breaks the current API. Best we can do is define it as `register:routeHandlerBlock:`